### PR TITLE
Popup with attributes for devices on map

### PR DESCRIPTION
**Pull request by ninioe**
When the user click on the marker, if the device has attributs a popup window is shown with the device's attributes list.

### CODE REVIEW

**Comment by tananaev** [comment]
Have you done jshint and jscs checks? Please do it BEFORE EVERY PULL REQUEST. I tell it to you every time, but you keep ignoring me.

**Comment by ninioe** [comment]
I am not ignoring you, those files already passed all the tests.
Somehow the 4 spaces in two lines where replaces with tabs by mistake.

also I found 4 errors in other files you have, you should check yourself too:
app\view\ReportController.js: line 55, col 50, Strings must use singlequote.
app\view\ReportController.js: line 65, col 56, Strings must use singlequote.
app\view\ReportController.js: line 110, col 69, Strings must use singlequote.
app\view\ReportController.js: line 120, col 62, Strings must use singlequote.

4 errors

**Comment by tananaev** [comment]
Please explain your changes to the `loadReport` method in `MapController.js`.

I give you the last chance to review your own pull request before giving it to me for review. Please check every line before saying that it's ready.

**Comment by ninioe** [comment]
Sorry for this, I didn't notice that this file was changed by someone else since I started working on it.
I didn't change this method, those lines where left by the older version of this file.
I will have to write and combine all my changes again on the latest versions of all those files I changed.

**Comment by ninioe** [comment]
Hi Anton,
I am trying to merge the 2 commits you did in the last hour but I'm getting a conflict in Style.js file
and I really don't know how to solve it without upsetting you.
I would really appreciate your assistance here.

Thanks
Erez

**Comment by tananaev** [comment]
Open any merge tool and resolve it. What exactly is the problem? If you don't have any merge tool, you should get one. It's also possible to merge it manually in any text editor if you refuse to use a proper tools.

Also, don't forget to review EVERYTHING before you tell me it's ready. I'm seriously saying that it's your last chance to get it through.

**Comment by ninioe** [comment]
Hi Anton,
Do you have any more comments on this pull request?

Thanks
Erez

File: web/l10n/en.json [hunk 1]
```diff
@@ -139,6 +140,7 @@
     "alarmOverspeed": "Overspeed Alarm",
     "alarmFallDown": "FallDown Alarm",
     "alarmLowBattery": "LowBattery Alarm",
+    "alarmMotion": "Motion Alarm",
```
Comment (tananaev) [inline web/l10n/en.json]: Why do you add this string here?

Comment (ninioe) [inline web/l10n/en.json]: I only added the "Zoom In" for the tool bar of the popup.

Comment (tananaev) [inline web/l10n/en.json]: That's because you haven't merged your changes properly.

**Pull request closed without merging**